### ISSUE TITLE
Remove announcement of Proof Assistants SE in about one month (mid-April).

### ISSUE THREAD

**Issue by Zimmi48** [rocq-prover/coq.github.io#195]
This is commit 4fed741c9970f3f4efccb6c9ee53cabbde11547e from #193.

**Comment by SkySkimmer** [comment]
ping?

**Comment by Zimmi48** [comment]
Yes, we should probably remove the announcement from the recent news section now. Instead, we could add a direct link from the "User community" section on the homepage.
